### PR TITLE
helm/prometheus: custom rules ConfigMap

### PR DESCRIPTION
**Pull request by MihaiAnei**
This fixes the problem introduced by #1096 

@gianrubio @andersflemmen are you okay with this approach?

### CODE REVIEW

**Comment by MihaiAnei** [comment]
Values:
```
rules:
  specifiedInValues: true
  additionalLabels: {}
  value:
    prometheus.rules: |-    
      groups:
      - name: prometheus.rules
        rules:
        - alert: PrometheusConfigReloadFailed
          expr: prometheus_config_last_reload_successful == 0
          for: 10m
          labels:
            severity: warning
          annotations:
            description: Reloading Prometheus' configuration has failed for {{$labels.namespace}}/{{$labels.pod}}
            summary: Reloading Promehteus' configuration failed
```
Output:
```
# Source: prometheus/templates/rules.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  labels:
    app: prometheus
    chart: prometheus-0.0.25
    heritage: Tiller
    prometheus: prometheus-test
    release: prometheus-test
  name: prometheus-prometheus-test-rules
data:
  prometheus.rules: |-
    groups:
    - name: prometheus.rules
      rules:
      - alert: PrometheusConfigReloadFailed
        expr: prometheus_config_last_reload_successful == 0
        for: 10m
        labels:
          severity: warning
        annotations:
          description: Reloading Prometheus' configuration has failed for {{$labels.namespace}}/{{$labels.pod}}
          summary: Reloading Promehteus' configuration failed
```
Values with custom labelling:
```
rules:
  specifiedInValues: true
  additionalLabels: 
    key1: value1
    key2: value2
    prometheus: coreos
  value:
    prometheus.rules: |-    
      groups:
      - name: prometheus.rules
        rules:
        - alert: PrometheusConfigReloadFailed
          expr: prometheus_config_last_reload_successful == 0
          for: 10m
          labels:
            severity: warning
          annotations:
            description: Reloading Prometheus' configuration has failed for {{$labels.namespace}}/{{$labels.pod}}
            summary: Reloading Promehteus' configuration failed
```
Output:
```
# Source: prometheus/templates/rules.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  labels:
    app: prometheus
    chart: prometheus-0.0.25
    heritage: Tiller
    prometheus: prometheus-test
    release: prometheus-test
    key1: value1
    key2: value2
    prometheus: coreos
    
  name: prometheus-prometheus-test-rules
data:
  prometheus.rules: |-
    groups:
    - name: prometheus.rules
      rules:
      - alert: PrometheusConfigReloadFailed
        expr: prometheus_config_last_reload_successful == 0
        for: 10m
        labels:
          severity: warning
        annotations:
          description: Reloading Prometheus' configuration has failed for {{$labels.namespace}}/{{$labels.pod}}
          summary: Reloading Promehteus' configuration failed
```

File: helm/prometheus/values.yaml [hunk 1]
```diff
@@ -139,6 +139,9 @@ rulesSelector: {}
 ##
 rules:
   specifiedInValues: true
+  ## What additional rules to be added to the ConfigMap
+  ## You can use this together with `rulesSelector`
+  additionalLabels: {}
```
Comment (gianrubio) [inline helm/prometheus/values.yaml]: please add a comment on additionalLabels value explaining how to use this variable

Comment (MihaiAnei) [reply helm/prometheus/values.yaml]: is this okay?

Comment (gianrubio) [reply helm/prometheus/values.yaml]: Could you also add in helm/kube-prometheus/Values.yaml ?

Comment (MihaiAnei) [reply helm/prometheus/values.yaml]: I forgot helm/prometheus is required in that chart. It should be okay now.

**Comment by gianrubio** [comment]
thanks @MihaiAnei

**Pull request merged**